src/lao.c

```diff
@@ -323,6 +353,14 @@ static int l_is_big_endian(lua_State *L)
 	return 1;
 }
 
+static int l_append_global_option(lua_State *L)
+{
+	const char *key = luaL_checkstring(L, 1);
+	const char *val = luaL_checkstring(L, 2);
```

Comment:
I think that might be going too far: My intention was just to have some mapping to `NULL`.
Perhaps I'll just leave this as it is (with string compulsory)